.github/workflows/build_portable_linux_python_packages.yml

```diff
@@ -11,19 +11,8 @@ on:
         description: Workflow run ID to download artifacts from
         type: string
         default: "17865324892" # TODO: default to the most recent successful run (using a script)
-      amdgpu_family:
-        type: choice
-        options:
-          - gfx101X-dgpu
-          - gfx103X-dgpu
-          - gfx110X-dgpu
-          - gfx1150
-          - gfx1151
-          - gfx120X-all
-          - gfx90X-dcgpu
-          - gfx94X-dcgpu
-          - gfx950-dcgpu
-        default: gfx94X-dcgpu
+      artifact_group:
+        type: string
```

Comment:
Can we keep a default for these, or at least add a description with a concrete example? I find it very valuable to be able to trigger these jobs with workflow dispatch and get some reasonable default (one of the most stable gfx targets that has test runners available).